Site/bc_weather_station_dashboard/website/templates/weather.html

```diff
@@ -176,7 +128,12 @@ <h2 class="card-text">12</h2>
         </div>
     </div>
 
-    <!-- The Modal -->
+    <!-- Swap pages button -->
+    <a href="{% url 'fire' %}">
+        <img id="swapPageIcon" src="{%  static 'flame_icon.svg' %}" alt="fire_icon" >
```

Comment:
Seems to still be an issue with image sizing showing as a very large button on the bottom left of screen

Reply:
can you try pressing ctrl + f5 to see if this goes away? or maybe cmd + f5 on mac